### ISSUE TITLE
Show Messages on All Monitors

### ISSUE THREAD

**Issue by cwarden** [dunst-project/dunst#45]
It would be nice if there was an option to show messages on all monitors.

**Comment by robbat2** [comment]
Did this ever change? I'd like some notifications to be shown on all monitors.

**Comment by JackofSpades707** [comment]
As would I. I would love either solution. The monitor I'm looking at or all of them to notify me.

**Comment by tsipinakis** [comment]
There is some discussion about moving dunst to a multiple-window system in #344 and it may be worth looking into this feature a bit more.

**Comment by egoldshtrom** [comment]
+1

I use `follow = keyboard`, which is great, but I would love the notification to show up on all monitors.

**Comment by PMunch** [comment]
I agree that this would be a nice feature. So far I've gone with a full-width notification in order to be able to notice it out of the corner of my eye if I'm currently looking at another monitor than the one with focus.

**Comment by prankousky** [comment]
+1 I use full width notifications as well, but working with multiple monitors still makes it hard to notice them sometimes.

**Comment by Phantas0s** [comment]
I would love this feature as well. I miss many notifications since I have 3 screens.

**Comment by pangteypiyush** [comment]
`follow = keyboard` works fine and I dont think this feature will ever get implemented.

**Comment by mickmattes** [comment]
@pangteypiyush 
Is it so difficult to understand for you? We don't want `follow = keyboard`. If you want to buy food, do you go home with toilet paper, cause it also fits into your mouth and eliminates hunger? Unbelievable.

**Comment by mickmattes** [comment]
Well if that's so, there are plenty of notification daemons out there. Who needs dunst?
[Desktop Notifications - Arch Linux](https://wiki.archlinux.org/index.php/Desktop_notifications#Standalone)

**Comment by tsipinakis** [comment]
> I dont think this feature will ever get implemented.

It'll get implemented if someone (including me) steps up to actually implement it - it's an open source project after all :).

My time for working on new features, especially as complex as this, is limited but I'm still very active for reviewing PRs and helping contributors if they need guidance on how to work on something.

**Comment by mitchellroe** [comment]
Thank you for `follow = keyoard`. I personally find the behavior a bit strange given the name of the parameter.

#### Steps to reproduce

Have OutputA and OutputB attached and configured. OutputA has focus. Get a notification. It shows up on OutputA. Move keyboard focus to OutputB.

#### Expected behavior

The notification moves to OutputB, i.e. it "follows" the keyboard focus.

#### Actual Behavior

The notification stays on OutputA. If additional notifications arrive before the timeout, those _also_ go to OutputA, regardless of where my keyboard focus is at the time.

**Comment by bynect** [comment]
> `follow = keyboard` works fine and I dont think this feature will ever get implemented.

Will surely get implemented when we move to multiple windows

**Comment by betodaviola** [comment]
`follow=keyboard` is indeed a very good compromise, but given the choice I would still prefer it on all monitors.
